internal/commands/inspect_builder.go

```diff
@@ -36,28 +36,33 @@ func InspectBuilder(logger logging.Logger, cfg config.Config, client PackClient)
 				imageName = args[0]
 			}
 
+			remoteOutput, remoteWarnings, remoteErr := inspectBuilderOutput(client, cfg, imageName, false)
+			localOutput, localWarnings, localErr := inspectBuilderOutput(client, cfg, imageName, true)
+
+			if remoteOutput == "(not present)" && localOutput == "(not present)" {
```

Comment:
I would feel more comfortable with a string comparison if it was a constant used inside the `inspectBuilderOutput` function and in this statement.

Alternatively, an additional return value similar to how maps returns a boolean value (`ok`) if it was found would work here.

No strong opinion either way but something should be changed to not just do arbitrary freeform string comparisons IMO.

Reply:
Did the 2nd option and added additional return value 'present'